bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -112,6 +118,93 @@ async def autoreview_status(self, ctx: Context) -> None:
         else:
             await ctx.send("Autoreview is currently disabled.")
 
+    @tasks.loop(hours=72)
+    async def track_forgotten_nominations(self) -> None:
+        """Track active nominations who are more than 2 weeks old."""
+        old_nominations = await self._get_forgotten_nominations()
+        untracked_nominations = await self._find_untracked_nominations(old_nominations)
+        for nomination, thread in untracked_nominations:
+            issue_created = await self._track_vote_in_github(nomination, thread.jump_url)
+            if issue_created:
+                await thread.starter_message.add_reaction(FLAG_EMOJI)
+
+    async def _get_forgotten_nominations(self) -> list[Nomination]:
+        """Get active nominations that are more than 2 weeks old."""
+        now = arrow.utcnow()
+        nominations = [
+            nomination
+            for nomination in await self.api.get_nominations(active=True)
+            if (now - nomination.inserted_at).days >= OLD_NOMINATIONS_THRESHOLD_IN_DAYS
```

Comment:
`nomination.inserted_at` is the datetime the nomination was created, rather than put up for review.

We want to use the date that the nomination message was posted up for review for this calculation.

Reply:
You're right.

This is fixed in babf90649df6acb5e8ebda28e15b4e4b1ee9e2d2 by relying on the thread_id to turn into into a valid datetime object & use it for comparison